.env.example

```diff
@@ -4,4 +4,6 @@ VITE_SAMS_AUTH_URL=
 VITE_SAMS_USER_INFO_URL=
 VITE_OAUTH_CALLBACK_URL=
 VITE_OAUTH_TOKEN_URL=
-VITE_UPLOAD_API_ENDPOINT=
\ No newline at end of file
+VITE_UPLOAD_API_ENDPOINT=
```

Comment:
Perhaps we should rework this env var into the `VITE_API_BASE_URL` variable and the `config/api.ts` file? Keep all of the api routes under one env variable.